ros/api/v0/recommendations.py

```diff
@@ -63,5 +63,4 @@ def get(self, host_id):
                   'meta': {'count': len(recommendations_list)}
             }
         else:
-            abort(404, message="host with id {} doesn't have any recommendation"
-                  .format(host_id))
+            recommendations_list = []
```

Comment:
@apuntamb - what is the expected return value in this case?

Reply:
return value should be common for both cases. added a commit for the same.